public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -237,6 +242,11 @@ export const LdapSettingsPage = () => {
       });
       reset(payload);
       reportInteraction('authentication_ldap_deleted');
+
+      // Delay redirect so the form state can update
+      setTimeout(() => {
+        locationService.push(`/admin/authentication`);
+      }, 300);
```

Comment:
This one too, I don't understand why we have to do this.

Reply:
Removed as requested